docs/provider/daemon/getting_started/overview.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Overview'
+sidebar_position: 1
+---
+
+A TDEX daemon can be served in one of the following ways:
+ * [Docker container](run_docker.md)
+ * [Standalone application](run_standalone.md)
+ * [TDEX Box](https://github.com/tdex-network/tdex-box) (docker-compose)
```

Comment:
Tdex Box would be part of the docker way.

I would eventually make a dedicated chapter

```suggestion

 Docker-compose box for production
 * [TDEX Box](https://github.com/tdex-network/tdex-box) (docker-compose)
```